lib/ssh_scan/banner.rb

```diff
@@ -29,6 +29,8 @@ def ssh_lib_guess()
         return SSHScan::SSHLib::CiscoSSH.new()
       when /ROS/i
         return SSHScan::SSHLib::ROSSSH.new()
+      when /DOPRASSH/i
```

Comment:
@claudijd Great :)

I downloaded a pretty big file from scans.io and wasn't able to find one, that's why I asked. Never mind, then.